pulpcore/tests/functional/api/test_tasks.py

```diff
@@ -179,3 +179,39 @@ def test_read_fields_created_resources_only(self):
             filtered_task['created_resources'],
             filtered_task,
         )
+
+
+class FilterTaskCreatedResourcesContentTestCase(unittest.TestCase):
+    """Perform filtering for contents of created resources."""
```

Comment:
If this work has a related pulp.plan.io issue it would be nice to add as part of the docstring. It is very helpful later on.

```suggestion
       
    """Perform filtering for contents of created resources.
    
       This test targets the following issue:
    * `Pulp #xxx <https://pulp.plan.io/issues/xxxx>`
    
    """
```